### PR TITLE
rename package to velocitas_sdk

### PR DESCRIPTION
**Pull request by MP91**
## Checklist - Manual tasks

* [ ] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: examples/dog-mode/src/vehicle_model/proto/hvac_pb2.py
Comment (doosuu) [inline examples/dog-mode/src/vehicle_model/proto/hvac_pb2.py:1]: This is a generated file and should not be modified. The package of the proto file did not change: https://github.com/eclipse/kuksa.val.services/blob/main/hvac_service/proto/sdv/edge/comfort/hvac/v1/hvac.proto

File: examples/dog-mode/src/vehicle_model/proto/hvac_pb2_grpc.py
Comment (doosuu) [inline examples/dog-mode/src/vehicle_model/proto/hvac_pb2_grpc.py:1]: Generated file

File: generate-grpc-stubs.sh [hunk 1]
```diff
@@ -20,8 +20,8 @@ echo "#######################################################"
 set -x
 
 ROOT_DIR=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )" )
+python3 -m grpc_tools.protoc -I ./velocitas_sdk/proto --grpc_python_out=./velocitas_sdk/proto --python_out=./velocitas_sdk/proto --mypy_out=./velocitas_sdk/proto ./velocitas_sdk/proto/**/*.proto
```
Comment (doosuu) [inline generate-grpc-stubs.sh:23]: Why remove the empty line? Looks weird in diff ;-)

**Review by doosuu — CHANGES_REQUESTED** [review]

File: examples/dog-mode/src/vehicle_model/proto/hvac_pb2.py
Comment (MP91) [reply examples/dog-mode/src/vehicle_model/proto/hvac_pb2.py:1]: you are totally right -> reverted

File: generate-grpc-stubs.sh (hunk 1 above)
Comment (MP91) [reply generate-grpc-stubs.sh:24]: reverted

**Review by doosuu — APPROVED** [review]
LGTM

**Review by doosuu — APPROVED** [review]
LGTM

File: generate-grpc-stubs.sh [hunk 2]
```diff
@@ -21,7 +21,8 @@ set -x
...
 
-python3 -m grpc_tools.protoc -I ./sdv/proto --grpc_python_out=./sdv/proto --python_out=./sdv/proto --mypy_out=./sdv/proto ./sdv/proto/**/*.proto
-mv -f ./sdv/proto/**/*.py ./sdv/proto
-mv -f ./sdv/proto/**/*.pyi ./sdv/proto
-sed -i -e 's/from sdv\..* import/from sdv.proto import/g' -e 's/import sdv\..*\./import sdv.proto./g' ./sdv/proto/*.py*
+python3 -m grpc_tools.protoc -I ./velocitas_sdk/proto --grpc_python_out=./velocitas_sdk/proto --python_out=./velocitas_sdk/proto --mypy_out=./velocitas_sdk/proto ./velocitas_sdk/proto/**/*.proto
+mv -f ./velocitas_sdk/proto/**/*.py ./velocitas_sdk/proto
+mv -f ./velocitas_sdk/proto/**/*.pyi ./velocitas_sdk/proto
+
+sed -i -e 's/from sdv\..* import/from velocitas_sdk.proto import/g' -e 's/import sdv\..*\./import velocitas_sdk.proto./g' ./velocitas_sdk/proto/*.py*
```
Comment (BjoernAtBosch) [inline generate-grpc-stubs.sh:28]: Hmh, basically this isn't correct: With that we pretend that the data broker stubs belong to the Velocitas SDK, but it isn't the case. Those stubs are only used by Velocitas and belong to data broker.
With this breaking renaming step, we now should move them into the right package/folder structure: `sdv/databroker/v1/...`
With that, we can - in a next step - easily create a Python package to be included (maybe generated on the fly)

Comment (MP91) [reply generate-grpc-stubs.sh:28]: hmm the path is actually coming from kuksa: https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker/proto/sdv/databroker/v1
Thats why I left it as it is...

**Pull request merged**